### PR TITLE
Fixed spelling

### PR DESCRIPTION
**Pull request by tarikozket**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ It contains two kinds of plugins to the [clang compiler](http://clang.llvm.org/)
 
 - analyzer plugins use the internal APIs of [clang analyzer](http://clang-analyzer.llvm.org/) to find bugs and report them;
 
-- frontend plugins process the syntax of source files directly to accomplish more general tasks; specifically, we have developed a clang-to-ocaml bridge to make code analyses easier.
+- frontend plugins process the syntax of source files directly to accomplish more general tasks; specifically, we have developed a clang-to-ocaml bridge to make code analyzes easier.
```
Comment (matbd) [inline README.md]: Afaik analyses is the plural of analysis: http://grammarist.com/spelling/analyses-vs-analyzes/

File: README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ It contains two kinds of plugins to the [clang compiler](http://clang.llvm.org/)
...
 
-Most of the plugins here have been written with iOS in mind. However different platforms may be considered in the future.
+Most of the plugins here have been written with iOS in mind. However, different platforms may be considered in the future.
```
Comment (matbd) [inline README.md]: sure

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Structure of the repository
 
 - [`libtooling`](https://github.com/facebook/facebook-clang-plugins/tree/master/libtooling) : frontend plugins (currently a clang-to-json AST exporter),
 
-- [`clang-ocaml`](https://github.com/facebook/facebook-clang-plugins/tree/master/clang-ocaml) : ocaml libraries to process the Json output of frontend plugins,
+- [`clang-ocaml`](https://github.com/facebook/facebook-clang-plugins/tree/master/clang-ocaml) : ocaml libraries to process the JSON output of frontend plugins,
```
Comment (matbd) [inline README.md]: ok

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Structure of the repository
 Quick start
 -----------
 
-The current version of the plugins requires the version 3.6.1 of the clang compiler, re-compiled from source.
+The current version of the plugins requires the version 3.6.1 of the clang compiler, re-compiled from the source.
```
Comment (matbd) [inline README.md]: "Compile from source" is the usual expression afaik, e.g. https://golang.org/doc/install/source

**Comment by tarikozket** [comment]
@matbd fixed. Thanks for your comments and feedback!

**Pull request merged**